azure-pipelines.yml

```diff
@@ -470,6 +470,14 @@ jobs:
       arguments: '--filter "Group=RawAssemblyEndToEndTests" --no-build'
       projects: |
         **\WebJobs.Script.Tests.Integration.csproj
+
+  - pwsh: |
```

Comment:
See other comment in "package.json" first. Otherwise, I think this should work for you:
```
- task: Npm@1
  displayName: 'npm ci'
  inputs:
    command: ci
    workingDir: sample/CustomHandlerRetry
```

Btw, `npm ci` is essentially the same as `npm install`, but it has some extra checks/validation and is designed to be run on build machines.